staging/src/k8s.io/client-go/copy.sh

```diff
@@ -65,71 +84,65 @@ echo "generating vendor/"
 GO15VENDOREXPERIMENT=1 godep save ./...
 popd > /dev/null
 
-echo "move to the client repo"
-# clean the ${CLIENT_REPO}
-ls "${CLIENT_REPO}" | { grep -v '_tmp' || true; } | xargs rm -rf
-mv "${CLIENT_REPO_TEMP}"/* "${CLIENT_REPO}"
-rm -r "${CLIENT_REPO_TEMP}"
-
 echo "moving vendor/k8s.io/kuberentes"
-cp -rn "${CLIENT_REPO}"/vendor/k8s.io/kubernetes/. "${CLIENT_REPO}"/
-rm -rf "${CLIENT_REPO}"/vendor/k8s.io/kubernetes
+cp -rn "${CLIENT_REPO_TEMP}"/vendor/k8s.io/kubernetes/. "${CLIENT_REPO_TEMP}"/
+rm -rf "${CLIENT_REPO_TEMP}"/vendor/k8s.io/kubernetes
 # client-go will share the vendor of the main repo for now. When client-go
 # becomes a standalone repo, it will have its own vendor
-mv "${CLIENT_REPO}"/vendor "${CLIENT_REPO}"/_vendor
-# remove the pkg/util/net/sets/README.md to silent hack/verify-munge-docs.sh
-# TODO: probably we should convert the README.md a doc.go
-find ./ -name "README.md" -delete
+mv "${CLIENT_REPO_TEMP}"/vendor "${CLIENT_REPO_TEMP}"/_vendor
 
 echo "rewriting Godeps.json"
-go run "${DIR}/godeps-json-updater.go" --godeps-file="${CLIENT_REPO}/Godeps/Godeps.json" --client-go-import-path="${CLIENT_REPO_FROM_SRC}"
+go run "${DIR}/godeps-json-updater.go" --godeps-file="${CLIENT_REPO_TEMP}/Godeps/Godeps.json" --client-go-import-path="${CLIENT_REPO_FROM_SRC}"
 
 echo "rewriting imports"
-grep -Rl "\"${MAIN_REPO_FROM_SRC}" ./ | grep ".go" | grep -v "vendor/" | xargs sed -i "s|\"${MAIN_REPO_FROM_SRC}|\"${CLIENT_REPO_FROM_SRC}|g"
+grep -Rl "\"${MAIN_REPO_FROM_SRC}" "${CLIENT_REPO_TEMP}" | \
+    grep "\.go" | \
+    grep -v "vendor/" | \
+    xargs sed -i "s|\"${MAIN_REPO_FROM_SRC}|\"${CLIENT_REPO_FROM_SRC}|g"
```

Comment:
All the grep patterns should exist, so I think it's fine.

I'm not sure if we can get rid of the pipe even if we use `find -exec`. But I'm not good at bash so I could be wrong.